Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -115,12 +115,15 @@ - (BOOL)textOf:(NSAttributedString*)newText equals:(NSAttributedString*)oldText{
     }
   }];
 
-  BOOL shouldFallbackToBareTextComparison =
-    [self.backedTextInputView.textInputMode.primaryLanguage isEqualToString:@"dictation"] ||
-    [self.backedTextInputView.textInputMode.primaryLanguage isEqualToString:@"ko-KR"] ||
-    self.backedTextInputView.markedTextRange ||
-    self.backedTextInputView.isSecureTextEntry ||
-    fontHasBeenUpdatedBySystem;
+  BOOL shouldFallbackDictation = [self.backedTextInputView.textInputMode.primaryLanguage isEqualToString:@"dictation"];
+  if (@available(iOS 16.0, *)) {
+    shouldFallbackDictation = self.backedTextInputView.dictationRecognizing;
+  }
+
+  BOOL shouldFallbackToBareTextComparison = shouldFallbackDictation ||
```

Comment:
Don't we have too many lines here? The diff is.. different 😄 https://github.com/facebook/react-native/commit/e8b4bb0684eed2acdb4648205893ee372f7ac1c7#diff-47e1ad911ac455bc7835356905ea0b6bf796d73af09734dc17a192f754eb5516R127

Reply:
@flodnv Thanks, I assume you mean we have a few more lines deleted and a few new lines added (although it is just a code formatting change), the code formatting change comes from this PR, https://github.com/facebook/react-native/commit /96027f787ade1a465569975059426d2c03ee5ad0#diff-4843243863da8dfee76a967566fcc2064856110f364a24a857d29d34fcdb4874L119-R126

But our Expensify/react-native has not been updated for this code format modification. For the convenience of updating the version and consistency in the future, I adopted the code format change of facebook